src/Renderer/Camera.cc

```diff
@@ -2,16 +2,16 @@
 
 namespace esp
 {
-  void renderer::Camera::set_orthographic(float left, float right, float top, float bottom, float near, float far)
+  void renderer::Camera::set_orthographic(float left, float right, float top, float bottom, float near_p, float far_p)
   {
```

Comment:
What's with this "_p"?

Reply:
It appears that 'far' and 'near' keywords are registered as macros on Windows and thus this code would not compile. I've added _p (as in far plane, near plane) to solve this issue.